src/screens/SavingsSheet.js

```diff
@@ -35,10 +35,7 @@ const SavingsSheet = () => {
   const supplyBalanceUnderlying = getParam('supplyBalanceUnderlying');
   const supplyRate = getParam('supplyRate');
 
-  const balance = convertAmountToNativeDisplay(
-    underlyingBalanceNativeValue,
-    nativeCurrency
-  );
+  const balance = nativeCurrencySymbol + underlyingBalanceNativeValue;
```

Comment:
This works fine for  the scenario $ 10 but  it looks like it would break scenarios where the currency is after the amount (ej 10 ETH)

Reply:
We need to separately clean this up, but currently we've got ETH's symbol set to Ξ. Normally for right-aligned currencies we use its `currency` value:
https://github.com/rainbow-me/rainbow/blob/637a865c1e34ff31742f8163af80a5ad527d23d5/src/helpers/utilities.js#L408-L416

That symbol isn't great but I think it looks better than putting ETH in the small text, and we're using it sparingly. Might make a better symbol at some point.

Reply:
Sounds good